src/types/Error.ts

```diff
@@ -0,0 +1,6 @@
+export enum Error {
+  NONE = '',
+  USERS = 'Something went wrong',
+  COMMENTS = 'Something went wrong',
```

Comment:
The error message for COMMENTS is too generic. Consider providing a more specific error message.